library/src/main/java/org/akop/ararat/view/CrosswordView.kt

```diff
@@ -2096,7 +2115,9 @@ class CrosswordView(context: Context, attrs: AttributeSet?) :
                         // Draw the unselected cell
                         val paint = when {
                             (v.markerDisplayMode and MARKER_SOLVED != 0) &&
-                                cell.isFlagSet(Cell.FLAG_SOLVED) -> v.solvedWordFillPaint
+                                    cell.isFlagSet(Cell.FLAG_SOLVED) -> if(index == sel.cell &&
+                                v.moveCursorToSolvedCell && !clearSelection) v.solvedOnSelectedPaint
```

Comment:
what `clearSelection` means?

Reply:
I think flag means clear selection for example in case of a new word so we don't need to mark cell in this case